lib/components/artist/shows/metadata.js

```diff
@@ -16,7 +16,7 @@ class Metadata extends React.Component {
         <Text style={styles.sansSerifText}>{this.props.show.partner.name.toUpperCase()}</Text>
         <Text style={styles.sansSerifText}>{this.showTypeString(this.props.show)}</Text>
         <SerifText style={styles.serifText}>{this.props.show.name}</SerifText>
-        { this.dateAndLocationString(this.props.show) }
+        { this.props.show.location ? this.dateAndLocationString() : null }
```

Comment:
Why not perform the location null check in that method as well? Probably reads better to have all conditionals in one place.